app/templates/index.hbs

```diff
@@ -24,9 +24,10 @@
             <div class="row">
                 <div class="text-center col-xs-12">
                     <p class="lead"> or </p>
-                    {{#link-to "submit" type="button" class="btn btn-success btn-lg" invokeAction=(action "click" "link" "Index - Add preprint")}}{{t "global.add_preprint"}}{{/link-to}}
-                    <div class="m-t-md">{{#link-to 'content' 'khbvy' invokeAction=(action "click" "link" "Index - See example")}}{{t "index.header.example"}}{{/link-to}}</div>
-
+                    {{#link-to (if theme.isProvider "provider.submit" "submit") type="button" class="btn btn-success btn-lg" invokeAction=(action "click" "link" "Index - Add preprint")}}
+                        {{t "global.add_preprint"}}
+                    {{/link-to}}
+                    <div class="m-t-md">{{#link-to (if theme.isProvider "provider.content" "content") 'khbvy' invokeAction=(action "click" "link" "Index - See example")}}{{t "index.header.example"}}{{/link-to}}</div>
```

Comment:
It's just hidden because the example link is hard coded to an OSF preprint, so this would take you out of the branded site, which I don't believe is the desired behavior. It should still be there for OSF Preprints. Need feedback from product on this one.